airbyte-webapp/Dockerfile

```diff
@@ -3,4 +3,4 @@ FROM nginx:1.19-alpine as webapp
 EXPOSE 80
 
 COPY build /usr/share/nginx/html
-COPY nginx/default.conf /etc/nginx/conf.d/default.conf
+COPY nginx/default.conf.template /etc/nginx/templates/default.conf.template
```

Comment:
why did you need to rename it?

Reply:
It is not a config itself anymore, it is a template.

According to docs of nginx Dockerfile 
```By default, this function reads template files in /etc/nginx/templates/*.template and outputs the result of executing envsubst to /etc/nginx/conf.d.```
So `envsubst` will replace `$TRACKING_STRATEGY` in the template with env variable and make proper nginx config.

Reply:
Got it.